lib/AS_BH1750/AS_BH1750A.cpp

```diff
@@ -457,7 +457,7 @@ void AS_BH1750A::defineMTReg(uint8_t val) {
     Serial.print("MGTreg low byte: ");
     Serial.println(loByte, BIN);
 #endif
-    write8(hiByte);
+    write8(loByte);  // As reported in https://github.com/hexenmeister/AS_BH1750/issues/2
```

Comment:
Hmm good catch.
Looks like the sensor may have been put into an incorrect sensitivity range?

Reply:
Well, I didn't calculate & compare to specs what it did send instead of what should have been sent, but it most likely wasn't very correct. But it still worked 😃.
I still think the response-speed is a bit weird (I mostly tested on an ESP32, and it takes several seconds after boot before valid data is coming from the sensor), but I bought this sensor (was ordering other stuff anyway) to try and fix the use with ESP32. Hopefully I find a proper use for it 😸.